### PR TITLE
Use github.com/logrusorgru/aurora instead of github.com/fatih/color

### PR DESCRIPTION
**Pull request by is2ei**
Fixed https://github.com/mingrammer/cfmt/issues/4

Passed all the tests.
```
$ git status
## use-aurora...origin/use-aurora
$ go test ./...
ok  	github.com/mingrammer/cfmt	(cached)
```

Could you review it, please?

### CODE REVIEW

**Comment by mingrammer** [comment]
The `color.Output` is an instance of `colorable.NewColorableStdout()` which is defined in `github.com/mattn/go-isatty`. It does not return just `os.Stdout` in Windows environment (https://github.com/fatih/color/blob/master/vendor/github.com/mattn/go-colorable/colorable_windows.go#L98).

So, you should use the `colorable.NewColorableStdout()`, not just with `os.Stdout`.

**Comment by is2ei** [comment]
@mingrammer Thanks ~~ 😄 
I updated it, could you review it?

File: cfmt.go [hunk 1]
```diff
@@ -1,211 +1,224 @@
...
 
-	"github.com/fatih/color"
+	"github.com/logrusorgru/aurora"
+	"github.com/mattn/go-colorable"
 )
 
+var (
+	// output defines the standard output of the print functions.
+	// By default, os.Stdout is used.
+	output = newStdout()
```
Comment (mingrammer) [inline cfmt.go]: `output = colorable.NewColorableStdout()` is more clear :)

`newStdout` is not required.

File: cfmt_test.go [hunk 2]
```diff
@@ -21,20 +19,25 @@ type (
 	SprintlnFunc func(...interface{}) string
 )
 
+const (
```
Comment (mingrammer) [inline cfmt_test.go]: Hm, I don't know it is a good idea that we declare the color code ourselves in test code. I think using color code from the color library is better. But `aurora` has no ANSI color codes .. 🤔

Or I think the color codes should be calculated using the `aurora.RedFg`, `aurora.Green` and so on.

**Review by mingrammer — CHANGES_REQUESTED** [review]

File: cfmt.go (hunk 1 above)
Comment (is2ei) [reply cfmt.go]: I agree and wil update it 👍

File: cfmt_test.go (hunk 2 above)
Comment (is2ei) [reply cfmt_test.go]: Ok! I will think about it 🤔

File: cfmt.go (hunk 1 above)
Comment (is2ei) [reply cfmt.go]: Done!

File: cfmt_test.go (hunk 2 above)
Comment (is2ei) [reply cfmt_test.go]: @mingrammer  I changed it to use `aurora.Color`. Could you check it?

File: cfmt.go [hunk 3]
```diff
@@ -1,211 +1,217 @@
...
 // Swarning returns yellow colored string in manner of fmt.Sprint
 func Swarning(a ...interface{}) string {
-	return color.New(color.FgYellow).Sprint(a...)
+	return fmt.Sprint(aurora.Brown(fmt.Sprint(a...)))
 }
 
 // Ferror prints red colored text in manner of fmt.Fprint
 func Ferror(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgRed).Fprint(w, a...)
+	n, err = fmt.Fprint(w, fmt.Sprint(aurora.Red(fmt.Sprint(a...))))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Fprint(w, fmt.Sprint(aurora.Red(fmt.Sprint(a...))))`

to

`fmt.Fprint(w, aurora.Red(fmt.Sprint(a...)))`

File: cfmt.go [hunk 4]
```diff
@@ -1,211 +1,217 @@
...
 // Serror returns red colored string in manner of fmt.Sprint
 func Serror(a ...interface{}) string {
-	return color.New(color.FgRed).Sprint(a...)
+	return fmt.Sprint(aurora.Red(fmt.Sprint(a...)))
 }
 
 // Fsuccessln prints green colored text in manner of fmt.Fprintln
 func Fsuccessln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgGreen).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, fmt.Sprint(aurora.Green(fmt.Sprintln(a...))))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Fprint(w, fmt.Sprint(aurora.Green(fmt.Sprintln(a...))))`

to

`fmt.Fprintln(w, aurora.Green(fmt.Sprint(a...)))`

File: cfmt.go [hunk 5]
```diff
@@ -1,211 +1,217 @@
...
 // Sinfoln returns cyan colored string in manner of fmt.Sprintln
 func Sinfoln(a ...interface{}) string {
-	return color.New(color.FgCyan).Sprintln(a...)
+	return fmt.Sprint(aurora.Cyan(fmt.Sprintln(a...)))
 }
 
 // Fwarningln prints yellow colored text in manner of fmt.Fprintln
 func Fwarningln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgYellow).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, fmt.Sprint(aurora.Brown(fmt.Sprintln(a...))))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Fprint(w, fmt.Sprint(aurora.Brown(fmt.Sprintln(a...))))`

to

`fmt.Fprintln(w, aurora.Brown(fmt.Sprint(a...)))`

File: cfmt.go [hunk 6]
```diff
@@ -1,211 +1,217 @@
...
 func Successln(a ...interface{}) (n int, err error) {
-	n, err = Fsuccessln(color.Output, a...)
+	n, err = Fsuccessln(output, a...)
 	return
 }
 
 // Ssuccessln returns green colored string in manner of fmt.Sprintln
 func Ssuccessln(a ...interface{}) string {
-	return color.New(color.FgGreen).Sprintln(a...)
+	return fmt.Sprint(aurora.Green(fmt.Sprintln(a...)))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Sprint(aurora.Green(fmt.Sprintln(a...)))`

to

`fmt.Sprintln(aurora.Green(fmt.Sprint(a...)))`

File: cfmt.go [hunk 7]
```diff
@@ -1,211 +1,217 @@
...
 }
 
 // Finfoln prints cyan colored text in manner of fmt.Fprintln
 func Finfoln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgCyan).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, aurora.Cyan(fmt.Sprintln(a...)))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Fprint(w, aurora.Cyan(fmt.Sprintln(a...)))`

to

`fmt.Fprintln(w, aurora.Cyan(fmt.Sprint(a...)))`

File: cfmt.go [hunk 8]
```diff
@@ -1,211 +1,217 @@
...
 func Infoln(a ...interface{}) (n int, err error) {
-	n, err = Finfoln(color.Output, a...)
+	n, err = Finfoln(output, a...)
 	return
 }
 
 // Sinfoln returns cyan colored string in manner of fmt.Sprintln
 func Sinfoln(a ...interface{}) string {
-	return color.New(color.FgCyan).Sprintln(a...)
+	return fmt.Sprint(aurora.Cyan(fmt.Sprintln(a...)))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Sprint(aurora.Cyan(fmt.Sprintln(a...)))`

to

`fmt.Sprintln(aurora.Cyan(fmt.Sprint(a...)))`

File: cfmt.go [hunk 9]
```diff
@@ -1,211 +1,217 @@
...
 func Warningln(a ...interface{}) (n int, err error) {
-	n, err = Fwarningln(color.Output, a...)
+	n, err = Fwarningln(output, a...)
 	return
 }
 
 // Swarningln returns yellow colored string in manner of fmt.Sprintln
 func Swarningln(a ...interface{}) string {
-	return color.New(color.FgYellow).Sprintln(a...)
+	return fmt.Sprint(aurora.Brown(fmt.Sprintln(a...)))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Sprint(aurora.Brown(fmt.Sprintln(a...)))`

to

`fmt.Sprintln(aurora.Brown(fmt.Sprint(a...)))`

File: cfmt.go [hunk 10]
```diff
@@ -1,211 +1,217 @@
...
 }
 
 // Ferrorln prints red colored text in manner of fmt.Fprintln
 func Ferrorln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgRed).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, aurora.Red(fmt.Sprintln(a...)))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Fprint(w, aurora.Red(fmt.Sprintln(a...)))`

to

`fmt.Fprintln(w, aurora.Red(fmt.Sprint(a...)))`

File: cfmt.go [hunk 11]
```diff
@@ -1,211 +1,217 @@
...
 func Errorln(a ...interface{}) (n int, err error) {
-	n, err = Ferrorln(color.Output, a...)
+	n, err = Ferrorln(output, a...)
 	return
 }
 
 // Serrorln returns red colored string in manner of fmt.Sprintln
 func Serrorln(a ...interface{}) string {
-	return color.New(color.FgRed).Sprintln(a...)
+	return fmt.Sprint(aurora.Red(fmt.Sprintln(a...)))
```
Comment (mingrammer) [inline cfmt.go]: Please change

`fmt.Sprint(aurora.Red(fmt.Sprintln(a...)))`

to

`fmt.Sprintln(aurora.Red(fmt.Sprint(a...)))`

**Review by mingrammer — CHANGES_REQUESTED** [review]

**Comment by mingrammer** [comment]
It might be the last request changes

File: cfmt_test.go (hunk 2 above)
Comment (mingrammer) [reply cfmt_test.go]: Using `aurora.Color` is good :)

**Comment by is2ei** [comment]
Thanks for your comment! I will update it.

**Comment by is2ei** [comment]
I updated it! Could you check it, please?

**Review by mingrammer — APPROVED** [review]

**Comment by mingrammer** [comment]
Sounds great!! Thanks for update the color library! :)

**Pull request merged**